Assets/Dev/Si Wai/Reward/Reward.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+
+[System.Serializable]
+public class Reward
+{
+    public string Title;
+    public float Weight;
+    public float Value;
+
+    public Reward(string title, float weight, float value) {
+        Title = title;
+        Weight = weight;
+        Value = value;
+    }
+
+    public void ActivateReward(Player player) {
```

Comment:
I am unsire if this is the right way to do it. Currently the Actions from Isis and Fabio are being finished up and finalized. I think we can really use those and then create a action for each stat.
Then, besides adding the Title and Weight, you also add the Action as one of the fields. 

This saves is from a switch case.
It also allows the designers to modify the rewards, instead of having to ask the devs to change it.
Note that a switch case based around strings is also a bad idea for multiple reasons.
1. If we want to change the title, we also have to change the switch case. Even though devs are supse to read the documentation before edititing, I am already 100% sure that pretty much no-one does that XD. so nobady knows that is going to be the case and they will break the code.
2. Furthermore, the designers also dont know that this is the case. And they see a little field called "Title" and think, hey cool, i can change the title. So its also a bad idea to couple them
3. switch cases with strings, or comparing strings as a means to action is a bad idea in general. It leaves room for typo's, as specially if the strings are not like 3 chars long, and also rely on capitalization. Secondly, string comparason is also relatively slow compared to comparing Enums or numbers.
 
 Anyways. I would leave it like this for now, leave the branch as it is for now untill the actions are merged. and then use that instead.